modules/virt-schedule-best-match-cpu-model-vms.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-scheduling-vms.adoc
+
+[id="virt-schedule-best-match-cpu-model-vms_{context}"]
+= Scheduling virtual machines with the best match for CPU model
+
+When a CPU model is not defined for a node and a virtual machine (VM), the CPU model for the VM is set to `host-model` and the VM is scheduled on the node that offers the best match for the VM's CPU model.
```

Comment:
CPU model will be set to a cluster defined default.

@stu-gott what default CPU do we set in kubevirt-config today?

Reply:
Sorry for the slow response. I wanted to make sure I was accurate:

We do not set a default CPU.

Reply:
Perhaps you were remembering this?

machine-type: pc-q35-rhel8.2.0

Reply:
Thanks - I just checked and we are setting them in the templates.

Reply:
Updating the Jira story with details. This paragraph was finetuned after consulting with Stu and Omer.